spec/features/cases/sar/case_approval.rb/sar_approval_spec.rb

```diff
@@ -68,4 +68,28 @@ def undo_de_escalate_case_as_disclosure_specialist(kase, case_list_item)
     de_escalate_case_as_disclosure_specialist(kase, case_list_item)
     undo_de_escalate_case_as_disclosure_specialist(kase, case_list_item)
   end
+
```

Comment:
No issues with the diff here, just ... `case_approval.rb` is the dir name? 😁 I'll change as part of CT-1755 which I'm basing on this branch. :)